packages/client/src/declarations/index.ts

```diff
@@ -1056,10 +1040,9 @@ function requestWithStateWrapper(
   return new Promise(async (resolve, reject) => {
     try {
       const data = await mainRequest
-      const userDetails = getUserDetails(getState())
-      if (
-        !FIELD_AGENT_ROLES.includes(userDetails?.systemRole as SystemRoleType)
-      ) {
+      const scopes = getScope(getState())
+      // @TODO: Check is record.register equal to `!FIELD_AGENT_ROLES.includes(userDetails?.systemRole as SystemRoleType)`
+      if (scopes?.includes('record.register')) {
```

Comment:
We also have a condition to `fetchAllDuplicateDeclarations` which has only matched `!FIELD_AGENT_ROLES.includes(userDetails?.systemRole as SystemRoleType)` previously. 

To be honest I'm not sure what this does yet, but this needs to be investigated